<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_CategoryMenu</class>
 <widget class="QDialog" name="Dialog_CategoryMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(74, 74, 74);
	color: #fff
}

 QPushButton{
	border: 1px solid rgb(182, 182, 182);
    border-radius: 5px;
    padding:8px;
}

QPushButton:hover {
	background-color: rgb(47, 47, 47);
}

QListView{
background-color: rgb(49, 49, 49);
border: 1px solid rgb(182, 182, 182);
border-radius: 5px;
padding:8px;
}

QListView:item{
border: none;
border-bottom: 1px solid rgb(189, 189, 189);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="listView_category">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="button_add">
           <property name="text">
            <string>Add +</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_edit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_delete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
